content-services/latest/install/ansible.md

```diff
@@ -61,110 +63,104 @@ all:
-| AOS | 1.4.1 | 1.4.0 |  1.4.0 | 1.3.1 |  |
-| GoogleDocs | 3.2.1 | 3.2.1 | 3.2.1 | 3.2.0 |  |
-| Digital Workspace | 2.7.0 | 2.6.0 | 2.1.0 | 2.6.0 | N/A |
-| Transform Router | 1.5.2 | 1.5.1 |1.3.2 | 1.5.1 | N/A |
-| Shared File Store | 0.16.1 | 0.16.1 | 0.13.0 | 0.16.1 | N/A |
-| Sync Service | 3.6.0 | 3.5.0 | 3.4.0 | 3.3.3.1 | N/A |
+| Component | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise | 7.0.N Enterprise | 6.2.N Enterprise | Community |
+|-|-|-|-|-|-|-|
+| OpenJDK | 17.0.3 | 11.0.13 | 11.0.13 | 11.0.13 | 11.0.13 | 11.0.13 |
+| Apache Tomcat | 9.0.59 | 9.0.59 | 9.0.59 | 9.0.59 | 8.5.72 | 9.0.59 |
+| PostgreSQL | 14.x | 13.x | 13.x | 13.x | 11.x | 13.x |
+| Apache ActiveMQ | 5.16.4 | 5.16.4 | 5.16.4 | 5.16.4 | 5.15.14 | 5.16.4 |
+| Repository | 7.3.0 | 7.2.1 | 7.1.1 | 7.0.1.4 | 6.2.2 | 7.1.1.2 |
+| Share | 7.3.0 | 7.2.1 | 7.1.1 | 7.0.1.4 | 6.2.2 | 7.1.1.2 |
+| Search Services | 2.0.5 | 2.0.4 | 2.0.2 | 2.0.1.1 | 1.4.3 | 2.0.3 |
+| Search Enterprise   | 3.2.0 | 3.1.1 | 3.1.1 | - | - | - |
+| All-In-One Transformation Engine | 3.0.0 | 2.5.7 | 2.5.6 | 2.3.10 | 2.5.6 | 2.5.7 |
+| AOS | 1.5.0 | 1.4.1 | 1.4.0 |  1.4.0 | 1.3.1 |  |
+| GoogleDocs | 3.3.0 | 3.2.2 | 3.2.1 | 3.2.1 | 3.2.0 |  |
+| Digital Workspace | 3.1.0 | 2.9.0 | 2.6.0 | 2.1.0 | 2.6.0 | N/A |
+| Transform Router | 2.0.0 | 1.5.3 | 1.5.1 |1.3.2 | 1.5.1 | N/A |
+| Shared File Store | 2.0.0 | 0.16.1 | 0.16.1 | 0.13.0 | 0.16.1 | N/A |
+| Sync Service | 3.8.0 | 3.6.0 | 3.5.0 | 3.4.0 | 3.3.3.1 | N/A |
 
 ## Set up Ansible
-Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe 
-how to configure a control node  with one of the many ways to set a python virtual environment. With python `virtualenvs` 
-you can install the exact same versions of Ansible we use when testing without impacting your system installation of python. 
-Doing so you're ensuring best chances of success.
+
+Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe how to configure a control node  with one of the many ways to set a python virtual environment. With python `virtualenvs` you can install the exact same versions of Ansible we use when testing without impacting your system installation of python. Doing so you're ensuring best chances of success.
 
 1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.0.0/alfresco-ansible-deployment-2.0.0.zip)
 
-2. If you're not working directly on the control node, transfer the ZIP file to the control node together with the 
-   SSH private key required to login to the target machines, and SSH into the machine
+2. If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the machine
```

Comment:
```suggestion
If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the control node
```

Reply:
I've changed the numbering as suggested.